test/examples/utils/plotting_helpers.jl

```diff
@@ -168,9 +170,11 @@ function plot_params!(sol, p; yscale = :log10, file_name = "box_model.pdf")
     for i in 1:n_dist
         ind_rng = get_dist_moments_ind_range(p.NProgMoms, i)
         for j in 1:size(params)[1]
-            moment_tuple = tuple(moments[j, ind_rng]...)
-            p.pdists[i] = CPD.update_dist_from_moments(p.pdists[i], moment_tuple)
-            params[j, ind_rng] = vcat(get_params(p.pdists[i])[2]...)
+            moms_tmp = moments[j, ind_rng]
+            pdist_tmp = CPD.update_dist_from_moments(p.pdists[i], ntuple(length(moms_tmp)) do i
+                moms_tmp[i]
+            end)
+            params[j, ind_rng] = vcat(get_params(pdist_tmp)[2]...)
```

Comment:
I think flatten is preferred over vcat - maybe you could use `rflatten` (I saw you added it in another file) here